CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # App Center SDK for React Native Change Log
 
+## Version 2.0.0
+
+Version 2.0.0 has a **breaking change**, it only supports Xcode 10.0.0+.
+
+### AppCenter Crashes
+
+#### iOS
+
+* **[Fix]** Print an error and return immediately when calling [MSCrashes notifyWithUserConfirmation:] with confirmation handlers not implemented.
```

Comment:
This is not Javascript syntax here. Please find the right syntax and update changelog (on docs or code).